config/app/commonConfigs.yaml

```diff
@@ -0,0 +1,46 @@
+objects:
+  console:
+    env:
+    - name: KIE_ADMIN_USER
+      value: adminUser
+    - name: KIE_ADMIN_PWD
+      value: "{{.AppPassword}}"
```

Comment:
These 3 passwords, when not specified, should all be generated to different values. Of course the same passwords should be reused in the server component for them to work together.

However, in the trial environment, when not specified, they should all be hardcoded to "RedHat"

Reply:
done... i hard-code the trial passwords within the CR, as its designed to do.